charts/misc-apps/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: misc-apps
 description: Argo CD app-of-apps config for miscellaneous small tools
 type: application
-version: 0.26.1
+version: 0.27.1
```

Comment:
Why `.1`? Shouldn’t that just become `0.27.0`?

Reply:
Sorry, my fault. Changed.